### ISSUE TITLE
Parity changed from working, to crashing on startup in Windows 10

### ISSUE THREAD

**Issue by Boterick** [paritytech/parity-ethereum#7859]
_Before filing a new issue, please **provide the following information**._

> I'm running: Alienware 18 dual 880 GTX SLI
>
> - **Which Parity version?**: 1.8.9
> - **Which operating system?**: Windows 10 pro
> - **How installed?**: via installer
> - **Are you fully synchronized?**: not with parity, crashes on startup
> - **Which network are you connected to?**: crashes on startup
> - **Did you try to restart the node?**: yes many times

Parity worked for a day or two, a few weeks ago on first install, achieving sync in about 6-8 hours. PC fans noticeably running at high speed during and after sync. PC shut down unexpectedly shortly after sync, apparently due to overheating whilst parity was running. First ever thermal shutdown for this high spec PC, normally used for benchmarking VR equipments and environments. On restart of PC after crash, attempt to restart parity resulted in parity failing to start. Tried uninstalling and reinstalling, still no joy. Since installed and ran Mist, and Geth to varying degress of success, but not well enough to carry out tasks needed on Ethereum chain. Returned now to trying again with new download of Parity, No change, still crashes on startup.

Further note: No further repeats of PC thermal shutdown experienced since Parity became unusable.

Attempting to run parity.exe from command prompt, including with admin rights, gives following output:

2018-02-11 16:00:01 UTC Starting Parity/v1.8.9-stable-98cccd5b2-20180201/x86_64-windows-msvc/rustc1.23.0
2018-02-11 16:00:01 UTC Keys path C:\Users\Magneto\AppData\Roaming\Parity\Ethereum\keys\Foundation
2018-02-11 16:00:01 UTC DB path C:\Users\Magneto\AppData\Local\Parity\Ethereum\chains\ethereum\db\906a34e69aec8c0d
2018-02-11 16:00:01 UTC Path to dapps C:\Users\Magneto\AppData\Roaming\Parity\Ethereum\dapps
2018-02-11 16:00:01 UTC State DB configuration: fast
2018-02-11 16:00:01 UTC Operating mode: passive
2018-02-11 16:00:01 UTC Couldn't read disk map: expected value at line 1 column 1
2018-02-11 16:00:01 UTC Configured for Foundation using Ethash engine

====================

stack backtrace:
   0:     0x7ff7c0c2cf2a - hid_error
   1:     0x7ff7c0c2d3c3 - hid_error
   2:     0x7ff7c0419e96 - <no info>
   3:     0x7ff7c0d014b3 - hid_error
   4:     0x7ff7c0d01331 - hid_error
   5:     0x7ff7c0d01212 - hid_error
   6:     0x7ff7c0d01185 - hid_error
   7:     0x7ff7c0d0a07f - hid_error
   8:     0x7ff7c0c50924 - hid_error
   9:     0x7ff7c0c5147b - hid_error
  10:     0x7ff7c0c2ad0f - hid_error
  11:     0x7ff7c0775899 - hid_error
  12:     0x7ff7c0817072 - hid_error
  13:     0x7ff7c02e9a9d - <no info>
  14:     0x7ff7c0305b2e - <no info>
  15:     0x7ff7c030cf0e - <no info>
  16:     0x7ff7c030e703 - <no info>
  17:     0x7ff7c0d02a52 - hid_error
  18:     0x7ff7c0d01762 - hid_error
  19:     0x7ff7c1024839 - hid_write
  20:     0x7ff84cb11fe4 - BaseThreadInitThunk

Thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: RlpExpectedToBeList', src\libcore\result.rs:906

This is a bug. Please report it at:

    https://github.com/paritytech/parity/issues/new

**Comment by 5chdn** [comment]
Duplicate of #7340

**Comment by Boterick** [comment]
problem unchanged, why closed?

**Comment by Tbaut** [comment]
Because a similar issue has already been open, please follow https://github.com/paritytech/parity/issues/7340

**Comment by niklasad1** [comment]
@Boterick Sorry for the inconvenience I have updated the reference to relevant open issue!

**Comment by niklasad1** [comment]
@5chdn @Tbaut I don't agree I think they are unrelated and therefore re-opened again!

**Comment by 5chdn** [comment]
Happy to have them both if you can distinguish them :)

**Comment by niklasad1** [comment]
@Boterick 

Hey, I can see that `unwrap` is triggered on `UntrustedRlp`
Can you provide the log in order to deduce which module fails by running the following:

```bash
parity.exe -l debug --log-file err.txt
```

**Comment by Boterick** [comment]
@niklasad1 Hi Niklas, thanks for your attention; when I try to run the command string above, I get the result below:

C:\Program Files\Parity Technologies\Parity>parity.exe -l debug --log-file err.txt

====================

stack backtrace:
   0:     0x7ff7ba9bcf2a - hid_error
   1:     0x7ff7ba9bd3c3 - hid_error
   2:     0x7ff7ba1a9e96 - <no info>
   3:     0x7ff7baa914b3 - hid_error
   4:     0x7ff7baa91331 - hid_error
   5:     0x7ff7baa91212 - hid_error
   6:     0x7ff7baa91185 - hid_error
   7:     0x7ff7baa9a07f - hid_error
   8:     0x7ff7b9ecb549 - <no info>
   9:     0x7ff7ba097783 - <no info>
  10:     0x7ff7ba09cf0e - <no info>
  11:     0x7ff7ba09e703 - <no info>
  12:     0x7ff7baa92a52 - hid_error
  13:     0x7ff7baa91762 - hid_error
  14:     0x7ff7badb4839 - hid_write
  15:     0x7ffc5a541fe4 - BaseThreadInitThunk

Thread 'main' panicked at 'Logger is initialized only once; qed: "Cannot write to log file given: err.txt"', src\libcore\result.rs:906

This is a bug. Please report it at:

    https://github.com/paritytech/parity/issues/new

**Comment by folsen** [comment]
@niklasad1 Did you make any progress on this?

**Comment by joshua-mir** [comment]
Hey @Boterick can you double check this is still an issue on the latest stable release? If so, can you try running parity as admin and seeing if this persists?  

I should note that in more recent releases of parity 
1) we don't distribute an installer but a binary, so you can either overwrite the existing parity.exe or just run parity.exe as is and
2) if you are expecting the UI, it has been deprecated since 1.10 and is not present in recent releases

Please comment if you would like this reopened.
